Add NFT How To (#1495)
iotaledger/iota-wiki: docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/nft/introduction.md

---
description: How to handle native NFTs on L2 and use them as ERC721
image: /img/logo/WASP_logo_dark.png
tags:
  - native 
  - NFT
  - EVM
  - how-to
---
import DocCardList from '@theme/DocCardList';

# Native NFT and ERC721

> The IOTA L1 has functionality to create NFTs, also called native NFTs. To use these NFTs on L2, you have
> a ERC20 contract called `ERC721NFTs` which contains all L1 NFTs owned by the chain. The following guides will show you how to [mint your own L1 NFT from L2](./mint-nft.md), and [use](./use-as-erc721.md) it with the `ERC721NFTs` contract.

Reviewer: ```suggestion
The IOTA L1 can create NFTs, also called native NFTs. To use these NFTs on L2, you have
an ERC20 contract called `ERC721NFTs`, which contains all L1 NFTs owned by the chain. The following guides will show you how to [mint your own L1 NFT from L2](./mint-nft.md) and [use](./use-as-erc721.md) it with the `ERC721NFTs` contract.
```
